src/betterproto/casing.py

```diff
@@ -120,3 +86,8 @@ def camel_case(value: str, strict: bool = True):
 
 def lowercase_first(value: str):
     return value[0:1].lower() + value[1:]
+
+
+def safen(value: str) -> str:
```

Comment:
this is an Improvement. Though could we have a clearer name? 

Maybe `make_safe` or `sanitize_name`